tests/rkt_tests.go

```diff
@@ -215,13 +215,17 @@ func importImageAndFetchHashAsGid(t *testing.T, ctx *testutils.RktRunCtx, img st
 	if err != nil || len(result) != 1 {
 		t.Fatalf("Error: %v\nOutput: %v", err, out)
 	}
+	err = child.Wait()
+	status := getExitStatus(err)
+	if status != 0 {
+		t.Logf("rkt terminated with unexpected status %d, expected %d\nOutput:\n%s", status, 0, child.Collect())
+		return "", fmt.Errorf("remote fetching failed, probably a network failure.")
```

Comment:
Could you include `img` in the error message? To know which image failed to be fetched.